crud/select/executor.lua

```diff
@@ -94,14 +100,21 @@ function executor.execute(space, index, filter_func, opts)
         end
 
         if tuple == nil then
-            return {}
+            return {
+                tuples = {},
+                stats = {
+                    tuples_fetched = 0,
+                    tuples_lookup = 0,
+                },
+            }
```

Comment:
Could be initialized in single place at start.

Also in TDG stats is collected in fiber.storage. It's probably less invasive way.

Reply:
It is less invasive, but it implies that everything runs in a single fiber. Isn't this too big of an assumption?

Reply:
Each your request work in separate fiber and at the end each fiber.storage is cleaned.

It's a way to keep some state in single place. TDG query executor is quite complex and it's quite hard to propagate state around all execution process. So this state was pinned in fiber.storage.

Probably here could be enough:
```lua
local stats = {
                    tuples_fetched = 0,
                    tuples_lookup = 0,
                }

...
inc_returned(stats)
...
inc_scanned(stats)

return tuples, stats
```

Reply:
Now everything that could be resolved with fiber.storage uses it.